src/effects/effectchainslot.cpp

```diff
@@ -44,6 +44,10 @@ EffectChainSlot::EffectChainSlot(EffectRack* pRack, const QString& group,
     m_pControlChainSuperParameter->set(0.0);
     m_pControlChainSuperParameter->setDefaultValue(0.0);
 
+    m_pControlChainSuperForceUpdate = new ControlObject(ConfigKey(m_group, "super_force_next_update"));
```

Comment:
I have not understand how and when this will be used. It looks to me like a miss-use of an control object. 
Maybe we miss a function in the control API?